### ISSUE TITLE
Show changes from calendar

### ISSUE THREAD

**Issue by georgehrke** [owncloud/activity#492]
from @Kawohl:

```
Hey Folks,

someone came up with this at the cebit.

It would be great to have changes in the calendar also in the activity stream of the specific users.... .

Cheers
```

https://github.com/owncloud/calendar/issues/355

**Comment by nickvergessen** [comment]
There is an API for it, but it will under go a major rework for 9.1 most likely.
So not sure if it's worth the trouble to implement it now.

Also after the API is done (it's in core) the calendar app should take care of it.

I will leave this issue open as a reminder and then create an issue in the calendar app to use the API, once it's done.

**Comment by georgehrke** [comment]
Well, with the calendar app only providing the GUI by now, it's not the right place in my opinion either. :confused: 

If both (caldav server and the new activity API) are in core, it should probably be implemented there as well. What do you think?

**Comment by nickvergessen** [comment]
Right, I forgot about that. But e.g. for tagging we do that as well. In terms of "if you don't use the UI, you don't want that either".
Other idea could be a "calendar activity app" or something like that. or we just put it into the activity app as a separated "bundle" (sub-app) which can be enabled/disabled simply with a checkbox by the admin.

**Comment by karlitschek** [comment]
We definitely should use activities and notification to show event from different apps like the calendar. This was the idea behind the activities in the first place :-)
@nickvergessen You are referring to the new external REST API right? I hope that the PHP doesn't change. And the old REST API should also still works.

**Comment by PVince81** [comment]
@felixheidecke

**Comment by mmluqman** [comment]
hi, i think this is very important in case of shared calendars. If there are any updates on this, please share ...
